samples/sample_decode/src/pipeline_decode.cpp

```diff
@@ -348,7 +348,7 @@ mfxStatus CDecodingPipeline::Init(sInputParams *pParams)
         switch (pParams->mode)
         {
         case MODE_PERFORMANCE:
-            m_memType = pParams->bUseHWLib? D3D9_MEMORY : SYSTEM_MEMORY;
+            m_memType = pParams->bUseHWLib ? D3D11_MEMORY : SYSTEM_MEMORY;
```

Comment:
yes, it looks like can affect linux, despite m_memType is not used directly for Linux, it influences to libType value
change it with defines, set SYSTEM_MEMORY if is not win

Reply:
It should be vaapi memory for performance more but we don't have separate VAAPI_MEMORY type so just re-using D3D9. So please D3D9_MEMORY for Linux in case of perf mode

Reply:
okey, done